packages/app-content-pages/src/shared/components/PageHeader/PageHeader.js

```diff
@@ -0,0 +1,65 @@
+import { AuthModal, ZooHeader } from '@zooniverse/react-components'
+import { useRouter } from 'next/router'
+import auth from 'panoptes-client/lib/auth'
+import { useContext, useState } from 'react'
+import { useUnreadMessages, useUnreadNotifications } from '@zooniverse/react-components/hooks'
+import { PanoptesAuthContext } from '../../contexts'
+
+function PageHeader() {
+  const [activeIndex, setActiveIndex] = useState(-1)
+  const { user } = useContext(PanoptesAuthContext)
+  const { data: unreadMessages }= useUnreadMessages(user)
+  const { data: unreadNotifications }= useUnreadNotifications(user)
```

Comment:
There's a bug here where these hooks make API requests even if you aren't logged in. It's not a bug in this PR, but it will need fixing at some point. It's probably happening because a logged-out user is an empty object, rather than null.

Reply:
I've opened #5705 to fix this in the hooks themselves.

Reply:
Hmm I still see API requests after the PR was merged even when I'm logged in! Will have to investigate more tomorrow...
<img width="300" alt="requests" src="https://github.com/zooniverse/front-end-monorepo/assets/23665803/61880524-55cd-4938-968a-734639367331">

Reply:
I get this same error in dev mode, because the content pages app uses `PANOPTES_ENV=production` in development. Either change the Panoptes environment to staging, or specify production in the URL in order to sign in to Panoptes production.
https://local.zooniverse.org:3000/about/team?env=production

A production build makes authenticated Talk requests without problems on http://local.zooniverse.org:3000/about/team, but you do get warned that you are sharing private information on an insecure connection.

Reply:
Compare the dev environment for the project app here:
https://github.com/zooniverse/front-end-monorepo/blob/19f7c72c88621fd5960be52ec6b2ccaa76f44d7a/packages/app-project/package.json#L10

with the dev environment for the content pages app here:
https://github.com/zooniverse/front-end-monorepo/blob/19f7c72c88621fd5960be52ec6b2ccaa76f44d7a/packages/app-content-pages/package.json#L10

Reply:
Alternatively, update the auth client to recognise the `PANOPTES_ENV` environment variable in the config here, then publish a minor version of the Panoptes client.

https://github.com/zooniverse/panoptes-javascript-client/blob/bef8c4465e12336afbfd56d4ee34088ec5b0f926/lib/config.js#L52-L64

Reply:
I'll add that there's nothing wrong with your code here. If you look through the config code for the old Panoptes JS Client, you'll see that it uses `NODE_ENV` to determine which API hosts to use. SO there's a mismatch when `NODE_ENV=development` and `PANOPTES_ENV=production`.

That's why `NODE_ENV` should really only be used to determine the build mode (development/production) and not the API host (development/staging/production.) In fact, 'staging' isn't a valid value for `NODE_ENV`, which should only be set to a webpack build mode.
https://webpack.js.org/configuration/mode/

Reply:
Oh! This makes sense, and I hadn't really thought about why notifications requests were failing in a dev env. Good to know it's not a bug, and I'll likely just update app-content-pages `yarn dev` to match app-project.